contracts/OMath.sol

```diff
@@ -0,0 +1,63 @@
+/**
+ * SPDX-License-Identifier: UNLICENSED
+ */
+pragma solidity 0.6.10;
+pragma experimental ABIEncoderV2;
+
+import {SafeMath} from "./packages/oz/SafeMath.sol";
+import {OtokenInterface} from "./interfaces/OtokenInterface.sol";
+import {ERC20Interface} from "./interfaces/ERC20Interface.sol";
+
+contract OMath {
```

Comment:
hmm can we call this something else? 

Some ideas: 
Converter
TokenTo1e18Converter